library_generation/generate_repo.py

```diff
@@ -77,9 +77,11 @@ def generate(
         config_path=generation_config_yaml,
         config=config,
         repository_path=repository_path,
-        target_library_names=target_library_names.split(",")
-        if target_library_names is not None
-        else target_library_names,
+        target_library_names=(
+            target_library_names.split(",")
```

Comment:
If I'm understanding correctly, this is splitting a String to a list by comma, and both the String field and the list field have the same name `target_library_names`. Can we reorganize the code and/or rename the fields to make it clear?

Reply:
This reformat is done by `black`. It updates to a new version so this line needs to reformat to pass python linter check.

This piece of code will be removed once `google-cloud-java` uses `entry_point.py`.

Can we leave it as-is?